src/main/java/project/bookstore/mapper/OrderMapper.java

```diff
@@ -0,0 +1,13 @@
+package project.bookstore.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import project.bookstore.config.MapperConfig;
+import project.bookstore.dto.order.OrderDto;
+import project.bookstore.model.Order;
+
+@Mapper(config = MapperConfig.class, componentModel = "spring", uses = OrderItemMapper.class)
```

Comment:
try without componentModel, because you have this parameter in MapperConfig.class (at least should have))) 
```suggestion
@Mapper(config = MapperConfig.class, uses = OrderItemMapper.class)
```